formulae/viewer/routes.py

```diff
@@ -96,112 +20,14 @@ def addviewer(objectId):
         objectId = str(editions[0].id)
     if current_app.config['nemo_app'].check_project_team() is True or objectId in current_app.config['nemo_app'].open_texts:
         template = {'manifest': 'viewer::miradorviewer.html'}
-        formulae = current_app.picture_file[objectId]
-        passage_data = ''
-        if request.args.get('embedded', False):
-            template = {'manifest': "viewer::multiviewermirador.html"}
-            passage_data = current_app.config['nemo_app'].r_passage(objectId, '1')
+        formulae = current_app.picture_file['manifest:' + re.match(r'.*(?=\.)',objectId).group(0)]
```

Comment:
In the `re.match` statement, there should be a space between the comma and `objectId`.